gateway/1.5/modules/ROOT/pages/flex-troubleshoot-logging.adoc

```diff
@@ -6,8 +6,32 @@ endif::[]
 
 When configuring logging for Anypoint Flex Gateway, you might encounter logging issues caused by incorrect directory permissions, and so on. The following topics provide possible causes for logging issues and describe how to troubleshoot:
 
+* <<third-party-service, Logs Do Not Appear in a Third-Party Logging Service>>
 * <<incorrect-permissions,Logs Do Not Appear when Running in a VM>>
 
+[[third-party-service]]
+== Logs Do Not Appear in a Third-Party Logging Service
+
+After applying a logging configuration YAML file, logs should begin appearing within a few minutes. If your logs do not appear in your third party logging service:
+
+. Ensure that your third-party service is running. How you do this is dependent on your chosen service.
+. If the service is running, check the runtime logs for potential configuration errors.
++
+You can view the runtime logs for both Flex Gateways running in Connected Mode and Local Mode by following the steps in xref:flex-local-view-logs.adoc[]. Additionally, for Flex Gateways running in Connected Mode, you can xref:flex-view-logs-in-monitoring.adoc[view runtime logs in Anypoint Monitoring].
++
+Potential configuration errors may appear similar to the following:
```

Comment:
To avoid ambiguity, I'd say:
"Configuration errors can be similar to the following example:"